### ISSUE TITLE
Add PeeringDB API

### ISSUE THREAD

**Issue by wallacemariadeandrade** [wallacemariadeandrade/BGPViewerTool#24]
API docs can be found at:

https://docs.peeringdb.com/
https://www.peeringdb.com/apidocs/

**Comment by wallacemariadeandrade** [comment]
Query to retrieve AS informations:

`https://www.peeringdb.com/api/net?asn__lte=16509&asn__gte=16509`

Output:

```
{
    "data": [
        {
            "id": 1418,
            "org_id": 1520,
            "name": "Amazon.com",
            "aka": "Amazon Web Services",
            "name_long": "",
            "website": "http://www.amazon.com",
            "asn": 16509,
            "looking_glass": "",
            "route_server": "",
            "irr_as_set": "AS-AMAZON",
            "info_type": "Enterprise",
            "info_prefixes4": 6000,
            "info_prefixes6": 2000,
            "info_traffic": "",
            "info_ratio": "Balanced",
            "info_scope": "Global",
            "info_unicast": true,
            "info_multicast": false,
            "info_ipv6": true,
            "info_never_via_route_servers": false,
            "ix_count": 129,
            "fac_count": 133,
            "notes": "AWS Peering: https://peering.aws/\n\n\nPeering requests:\n\nWhen submitting a peering request, please address the specific regional contact listed below for the location of your request (i.e. peering requests for London should use peering-emea@amazon.com while peering requests for Singapore should use peering-apac@amazon.com). This will ensure your request is processed and addressed in a timely fashion. Please do not copy contacts not meant for peering policy in the location of your request.\n\n-----\n\nOperational issues:\n\nIf you experience connectivity issues to Amazon, please first visit: http://ec2-reachability.amazonaws.com/.\nWhen contacting our Operational alias below, please include trace/debug information (source and destination address) and details for the affected prefixes. This will reduce troubleshooting time. Please do not contact policy addresses for operational support.\n\n----\n\nInformational:\n\nFor details about our network, routing and interconnection options, please see https://peering.aws.\nA list of our IP ranges with source location information is maintained here: https://ip-ranges.amazonaws.com/ip-ranges.json.\nPlease use the Maintenance alias to notify us regarding maintenance activities on your end and do not copy other aliases into these inquiries.  \n\n----\n\nRPKI:\n\nAWS drops all routes deemed to be RPKI invalid on incoming eBGP sessions.",
            "netixlan_updated": "2021-12-06T16:51:30.597577Z",
            "netfac_updated": "2021-11-23T16:21:38.116605Z",
            "poc_updated": "2020-12-01T12:29:55Z",
            "policy_url": "https://aws.amazon.com/peering/policy",
            "policy_general": "Selective",
            "policy_locations": "Preferred",
            "policy_ratio": false,
            "policy_contracts": "Not Required",
            "allow_ixp_update": false,
            "created": "2007-12-20T16:39:54Z",
            "updated": "2021-10-20T12:54:19Z",
            "status": "ok"
        }
    ],
    "meta": {}
}
```

When there aren't data about provided AS the result is as follows:

```
{
    "data": [],
    "meta": {}
}
```

**Comment by wallacemariadeandrade** [comment]
Query to retrieve IXs (I):

`https://www.peeringdb.com/api/netfac?net__asn__lte=53181&net__asn__gte=53181`

Output (I):
```
{
    "data": [
        {
            "id": 50916,
            "name": "Equinix SP4 - São Paulo",
            "city": "Barueri",
            "country": "BR",
            "net_id": 10605,
            "fac_id": 165,
            "local_asn": 53181,
            "created": "2021-08-27T19:33:35Z",
            "updated": "2021-08-27T19:33:35Z",
            "status": "ok"
        },
        {
            "id": 36065,
            "name": "Level(3) Rio de Janeiro",
            "city": "Rio de Janeiro",
            "country": "BR",
            "net_id": 10605,
            "fac_id": 1374,
            "local_asn": 53181,
            "created": "2019-06-17T18:46:24Z",
            "updated": "2019-06-17T18:46:24Z",
            "status": "ok"
        },
        {
            "id": 50915,
            "name": "Equinix RJ1 - Rio de Janeiro",
            "city": "Rio de Janeiro",
            "country": "BR",
            "net_id": 10605,
            "fac_id": 1586,
            "local_asn": 53181,
            "created": "2021-08-27T19:33:18Z",
            "updated": "2021-08-27T19:33:18Z",
            "status": "ok"
        },
        {
            "id": 21338,
            "name": "LNCC",
            "city": "Rio de Janeiro",
            "country": "BR",
            "net_id": 10605,
            "fac_id": 1641,
            "local_asn": 53181,
            "created": "2016-08-18T17:27:21Z",
            "updated": "2016-08-18T17:27:21Z",
            "status": "ok"
        },
        {
            "id": 24890,
            "name": "Equinix RJ2 - Rio de Janeiro",
            "city": "Rio de Janeiro",
            "country": "BR",
            "net_id": 10605,
            "fac_id": 1899,
            "local_asn": 53181,
            "created": "2017-06-06T19:20:01Z",
            "updated": "2017-06-06T19:20:01Z",
            "status": "ok"
        },
        {
            "id": 51143,
            "name": "Teleporto Rio de Janeiro",
            "city": "Rio de Janeiro",
            "country": "BR",
            "net_id": 10605,
            "fac_id": 2124,
            "local_asn": 53181,
            "created": "2021-09-10T15:44:55Z",
            "updated": "2021-09-10T15:44:55Z",
            "status": "ok"
        },
        {
            "id": 50918,
            "name": "Level(3) Sao Paolo",
            "city": "Cotia",
            "country": "BR",
            "net_id": 10605,
            "fac_id": 4490,
            "local_asn": 53181,
            "created": "2021-08-27T19:43:16Z",
            "updated": "2021-08-27T19:43:17Z",
            "status": "ok"
        },
        {
            "id": 50917,
            "name": "Datacenter Grupo Globo (PIX Globo.com IX-RJ)",
            "city": "Rio de Janeiro",
            "country": "BR",
            "net_id": 10605,
            "fac_id": 5375,
            "local_asn": 53181,
            "created": "2021-08-27T19:40:01Z",
            "updated": "2021-08-27T19:40:01Z",
            "status": "ok"
        }
    ],
    "meta": {}
}
``` 

Query to retrieve IXs (II):

`https://www.peeringdb.com/api/netixlan?asn__lte=53181&asn__gte=53181`

Output (II):

```
{
    "data": [
        {
            "id": 30017,
            "net_id": 10605,
            "ix_id": 171,
            "name": "IX.br (PTT.br) São Paulo: ATM/MPLA",
            "ixlan_id": 171,
            "notes": "",
            "speed": 100000,
            "asn": 53181,
            "ipaddr4": "187.16.222.222",
            "ipaddr6": "2001:12f8::222:222",
            "is_rs_peer": true,
            "operational": true,
            "created": "2016-08-18T17:22:36Z",
            "updated": "2021-08-26T12:40:18Z",
            "status": "ok"
        },
        {
            "id": 68197,
            "net_id": 10605,
            "ix_id": 171,
            "name": "IX.br (PTT.br) São Paulo: ATM/MPLA",
            "ixlan_id": 171,
            "notes": "",
            "speed": 100000,
            "asn": 53181,
            "ipaddr4": "187.16.212.67",
            "ipaddr6": "2001:12f8::212:67",
            "is_rs_peer": true,
            "operational": true,
            "created": "2021-06-24T19:29:15Z",
            "updated": "2021-08-19T14:21:49Z",
            "status": "ok"
        },
        {
            "id": 30018,
            "net_id": 10605,
            "ix_id": 177,
            "name": "IX.br (PTT.br) Rio de Janeiro: ATM/MPLA",
            "ixlan_id": 177,
            "notes": "",
            "speed": 20000,
            "asn": 53181,
            "ipaddr4": "45.6.52.153",
            "ipaddr6": "2001:12f8:0:2::153",
            "is_rs_peer": true,
            "operational": true,
            "created": "2016-08-18T17:23:58Z",
            "updated": "2017-12-27T11:42:33Z",
            "status": "ok"
        },
        {
            "id": 35091,
            "net_id": 10605,
            "ix_id": 177,
            "name": "IX.br (PTT.br) Rio de Janeiro: ATM/MPLA",
            "ixlan_id": 177,
            "notes": "",
            "speed": 100000,
            "asn": 53181,
            "ipaddr4": "45.6.52.210",
            "ipaddr6": "2001:12f8:0:2::210",
            "is_rs_peer": true,
            "operational": true,
            "created": "2017-06-06T19:21:09Z",
            "updated": "2021-03-03T15:24:30Z",
            "status": "ok"
        },
        {
            "id": 44890,
            "net_id": 10605,
            "ix_id": 177,
            "name": "IX.br (PTT.br) Rio de Janeiro: ATM/MPLA",
            "ixlan_id": 177,
            "notes": "",
            "speed": 100000,
            "asn": 53181,
            "ipaddr4": "45.6.53.32",
            "ipaddr6": "2001:12f8:0:2::53:32",
            "is_rs_peer": true,
            "operational": true,
            "created": "2018-10-31T18:09:28Z",
            "updated": "2020-03-06T02:02:12Z",
            "status": "ok"
        },
        {
            "id": 67244,
            "net_id": 10605,
            "ix_id": 706,
            "name": "IX.br (PTT.br) Vitória: ATM/MPLA",
            "ixlan_id": 706,
            "notes": "",
            "speed": 10000,
            "asn": 53181,
            "ipaddr4": "187.16.194.72",
            "ipaddr6": "2001:12f8:0:17::72",
            "is_rs_peer": true,
            "operational": true,
            "created": "2021-05-17T15:16:54Z",
            "updated": "2021-05-17T15:17:23Z",
            "status": "ok"
        }
    ],
    "meta": {}
}
```

**Comment by wallacemariadeandrade** [comment]
Successfully implemented at #37
